### PR TITLE
T17: retrieve a comment by ID

### PR DESCRIPTION
**Pull request by adrianovi**

### CODE REVIEW

File: src/main/java/api/methods/APICategoriesMethods.java [hunk 1]
```diff
@@ -6,12 +6,15 @@
...
 import io.restassured.response.Response;
+import utils.LoggerManager;
 import utils.StringManager;
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 
 public class APICategoriesMethods {
+    public static final LoggerManager log = LoggerManager.getInstance();
```
Comment (Agustin-Mediotti) [inline src/main/java/api/methods/APICategoriesMethods.java:17]: The instances should be private

File: src/main/java/api/methods/APICommentsMethods.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import io.restassured.response.Response;
+import utils.LoggerManager;
+import utils.StringManager;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class APICommentsMethods {
+    public static final LoggerManager log = LoggerManager.getInstance();
```
Comment (Agustin-Mediotti) [inline src/main/java/api/methods/APICommentsMethods.java:17]: The same here

Comment (adrianovi) [reply src/main/java/api/methods/APICommentsMethods.java]: Done

File: src/main/java/api/methods/APICommentsMethods.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import utils.LoggerManager;
+import utils.StringManager;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class APICommentsMethods {
+    private static final LoggerManager log = LoggerManager.getInstance();
+    public static final APIManager apiManager = APIManager.getInstance();
```
Comment (Agustin-Mediotti) [inline src/main/java/api/methods/APICommentsMethods.java:18]: this ones too

**Review by Agustin-Mediotti — COMMENTED** [review]
Plz change all public static final instantiations to **private**

File: src/main/java/api/methods/APICommentsMethods.java (hunk 3 above)
Comment (adrianovi) [reply src/main/java/api/methods/APICommentsMethods.java]: Done for all

**Review by Agustin-Mediotti — APPROVED** [review]
Good Job 🧉

**Pull request merged**